tests/test_csc.py

```diff
@@ -14,10 +18,13 @@ def __init__(self):
         index = 0  # next(index_gen)
         salobj.test_utils.set_random_lsst_dds_domain()
         self.csc = SchedulerCSC(index=index)
+        self.csc.parameters.mode = 'DRY'  # need to set this to allow tests
         self.remote = salobj.Remote(SALPY_Scheduler, index)
 
 # FIXME: probably need to take care of LSST_DDS_DOMAIN
 
+
```

Comment:
Yes you should set LSST_DDS_DOMAIN for each test -- in the setUp method (which you can add to each test class) call `salobj.test_utils.set_random_lsst_dds_domain()`.